quupe-RN/js/screens/SignIn/SignInContainer.js

```diff
@@ -2,10 +2,31 @@ import React, { Component } from 'react';
 import SignIn from './SignIn';
 
 class SignInContainer extends Component {
+  constructor() {
+    super();
+    this.state = {
+      username: '',
+      password: ''
+    }
+  }
+
```

Comment:
We are not going to use the component state to handle a user login. The server is up and running for auth. Please make changes to login so a user can log in and signup using our GraphCool server. Thank you.